ravenframework/PluginManager.py

```diff
@@ -91,7 +91,12 @@ def finishLoadPlugin(name):
     @ Out, available, bool, if True then plugin is available to use
   """
   if name not in _delayedPlugins:
-    return False
+    #Try to find module in system (such as from pip install)
+    spec = importlib.util.find_spec(name)
+    if spec is None:
+      return False
+    plugin = importlib.util.module_from_spec(spec)
+    _delayedPlugins[name] = (spec, plugin, False)
```

Comment:
Note that loadEntities will check that PluginBase.PluginBase is used as a base class for some class in this before it actually gets used as a plugin.